pkg/providers/oracle/objectstore/objectstore.go

```diff
@@ -0,0 +1,347 @@
+package objectstore
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/banzaicloud/pipeline/auth"
+	"github.com/banzaicloud/pipeline/pkg/objectstore"
+	model "github.com/banzaicloud/pipeline/pkg/providers/oracle/model/objectstore"
```

Comment:
Not sure if this alias is necessary, once files are moved to the proper packages.

Reply:
In the current structure the `objectstore` model should be in the `oracle/model` package anyway, but putting it there would cause cyclic imports right now.